### ISSUE TITLE
How to set the print size of an image?

### ISSUE THREAD

**Issue by kitoide** [mtmsuhail/ESC-POS-USB-NET#7]
When an image is printed it occupies the entire paper, how can I control the size?

**Comment by Laurens90** [comment]
I don't think it is possible. A bitmap has a certain size, so you will have to adjust your original image to the correct size. 
You can give a size of the QR-code, though.

**Comment by kitoide** [comment]
> I don't think it is possible. A bitmap has a certain size, so you will have to adjust your original image to the correct size.
> You can give a size of the QR-code, though.

I don't know if it was on the printer that I tried, but even reducing the original image he stretched it across the width of the paper

**Comment by aphestark** [comment]
for now that impossible, i have a tricky way using white space around image to control image size
